packages/components/src/components/form/Select/index.tsx

```diff
@@ -49,18 +54,30 @@ const selectStyle = (
             },
         };
     },
+    valueContainer: (base: Record<string, any>) =>
```

Comment:
since our original Select component didn't have any custom styles for valueContainer I would prefer to apply these custom settings only if it is  the "CleanSelect" variant, just to be sure we won't break too many things. BUT if you promise it'll be fine then I am gonna trust you 😁